### PR TITLE
Allow work objects to be created

### PR DESCRIPTION
**Pull request by jcoyne**
Screen Shot 2020-09-12 at 9 08 40 PM

### CODE REVIEW

**Review by mjgiarlo — APPROVED** [review]
Looks great!

**Comment by jcoyne** [comment]
@mjgiarlo this is just here so that we can see if Motion is going to be a workable solution. So far this leads me to believe there are complications ( https://github.com/unabridged/motion/issues/53 ). After we merge this PR, we can evaluate if stimulus-reflex might work or if we need to make this form without using the Rails helpers.

**Pull request merged**
